src/common/types.ts

```diff
@@ -12,15 +12,24 @@ export interface RGB_obj {
     B: number,
 }
 
-export type MethodIndex = '-1'|'0'|'1'|'2'|'3'
+export type MethodIndex = '-1' | '0' | '1' | '2' | '3'
 export interface ConfiguredPages {
     [key: string]: MethodIndex
 }
 export interface ConfiguredTabs {
     [key: number]: MethodIndex
 }
+
+export const ActivationMode = {
+    Off: 'off',
+    On: 'on',
+    'System Theme': 'system-theme',
+} as const;
+
+export type ActivationModeType = typeof ActivationMode[keyof (typeof ActivationMode)];
+
 export interface AddonOptions {
-    enabled: boolean,
+    activation: ActivationModeType,
```

Comment:
May be name the option something like `enable_policy`?

Reply:
pushed and update